tests/test_rotmat.py

```diff
@@ -44,6 +44,7 @@ def test_maths(self):
         assert v1 - v2 == Vector3(0, -1, -6)
         assert (v1 * 3) == Vector3(3, 6, -9)
         assert v1 * v2 == -2
+        assert v2 / 2.0 == Vector3(0.5, 1.5, 1.5)
```

Comment:
Also, add tests for true_div?

Reply:
this is already true_div, you mean floor_div  ?